source/WorkingPractices/gh_authorisation.rst

```diff
@@ -3,10 +3,107 @@
 Setting Up Github Authorisations
 ================================
 
-A brief overview of how to set up authorisations to gh for Sim Sys or links to
-relevant instructions.
+Initial Setup
+-------------
 
-* ssh keys
-* verified commits
-* gh command line
-* Maybe more?
+.. note::
+
+    The topics on this page are all more completely covered in other
+    documentation, which should be used first when finding any issues. Common
+    steps required to begin working with simulation systems repositories are
+    described here however. We will endeavour to provide links to other
+    documentation throughout.
+
+All users looking to interact with the simulation system repositories, will need
+to create a `github account
+<https://docs.github.com/en/get-started/start-your-journey/creating-an-account-on-github>`_.
+
+`Multi Factor Authentication
+<https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication>`_
+should also be enabled. This will be a requirement to accesses certain
+repositories, but is encouraged for all users.
+
+.. admonition:: todo
+
+    External access to MO org
+
+.. important::
+
+    To access some private repositories (eg.
+    UM) will require access to the MetOffice github organisation. Internal Met
+    Office employees can request this through ServiceNow.
+
+Before starting to use github, you will also need to configure your `user name
+<https://docs.github.com/en/get-started/git-basics/setting-your-username-in-git?platform=linux>`_
+and `commit email address
+<https://docs.github.com/en/account-and-profile/how-tos/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address>`_.
```

Comment:
Hmm... considering most of our repositories will be public facing, I am not sure about this. Some users may not want to expose their email addresses. So I think we should provide additional information on how to do that:

- Navigate to [Email settings](https://github.com/settings/emails)
- Select **Keep my email addresses private**. This should generate a unique noreply email address for you, which will look like `ID+username@users.noreply.github.com`
- Update gitconfig: `git config --global user.email "ID+username@users.noreply.github.com"`
- Verify: `git config --global user.email`